### DISCUSSION TITLE
It's been 24 days since I raised my ticket and I haven't got a single response from support

### DISCUSSION THREAD

**Discussion by Kareem-dev** [community/community#130274] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

As the title states, I raised a ticket about 24 days ago (5th of June) (ticket ID: 2827874) regarding the sudden and random suspension of my account (@Kareem01001011) for no apparent reason. I raised this reinstatement ticket as soon as I was suspended but I haven't gotten a single response ever since I raised the ticket. I made a discussion post 7 days after the suspension and I was basically told that I'll have to wait until they respond, but this is getting a little out of hand now.

I would like to get an update regarding my situation as soon as possible, and if possible, escalate my ticket/request since it has taken too long and it's causing me a lot of stress and trouble with my work.

Thank you for your understanding.

**Comment by Anirban2099** [comment]
Response #

**Comment by Kareem-dev** [comment]
It's been a month now... Any updates?
